functions/src/process-email-queue.ts

```diff
@@ -80,6 +80,16 @@ const processEmail = async (emailSnapshot: DocumentSnapshot) => {
     ...message,
     to: config.features.emails.recipient_override || message.to
   });
+
+  await db
+    .collection(EMAILS_TO_SEND_COLLECTION_NAME)
+    .doc(emailSnapshot.id)
```

Comment:
I think you can do `emailSnapshot.ref.update(...)`

Reply:
I was looking for something like this, thanks